testsuites/sparql/pom.xml

```diff
@@ -96,6 +96,11 @@
 			<artifactId>rdf4j-repository-contextaware</artifactId>
 			<version>${project.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>${project.groupId}</groupId>
+			<artifactId>rdf4j-repository-manager</artifactId>
+			<version>${project.version}</version>
+		</dependency>
```

Comment:
Ugh - I should take this out again, I put this in to try something but went with a different approach instead.